dev-tools/packaging/packages.yml

```
@@ -449,6 +449,16 @@ shared:
       from: 'arm64v8/centos:7'
       buildFrom: 'arm64v8/centos:7'
 
+  - &cloud_agent_docker_spec
```

I'd keep namespace prefix as is, using `cloud` as part of the suffixes
```suggestion
  - &agent_docker_cloud_spec
```